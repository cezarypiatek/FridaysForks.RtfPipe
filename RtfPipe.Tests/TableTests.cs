using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace RtfPipe.Tests;

[TestClass]
public class TableTests
{
  [TestMethod]
  public void marge_scenario_1() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_2() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_3() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_4() => this.VerifyCurrentScenario();

  [TestMethod]
  public void marge_scenario_5() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_6() => this.VerifyCurrentScenario();
  
  [TestMethod]
  public void marge_scenario_7() => this.VerifyCurrentScenario();
}
