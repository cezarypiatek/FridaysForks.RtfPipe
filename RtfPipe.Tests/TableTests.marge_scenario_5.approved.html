<div style="font-size:12pt;font-family:&quot;Times New Roman&quot;, serif;">
    <table style="border-spacing:0;font-size:inherit;box-sizing:border-box;margin:0 0 0 0.3px;">
        <colgroup>
            <col style="width:42.7px;" />
            <col style="width:102.7px;" />
            <col style="width:138.3px;" />
            <col style="width:85.1px;" />
        </colgroup>
        <tr style="text-align:left;height:20px;">
            <td style="text-align:center;font-family:Calibri, sans-serif;background:#D9D9D9;border:1px solid;padding:0 4.7px;">
                <strong>Nr
                </strong>
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;background:#D9D9D9;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <strong>Nazwa pom.
                </strong>
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;background:#D9D9D9;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <strong>Rodzaj posadzki
                </strong>
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;background:#D9D9D9;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <strong>Powierzchnia
                </strong>
            </td>
        </tr>
        <tr style="text-align:left;height:20px;">
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">1/1
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Korytarz
            </td>
            <td style="text-align:center;font-family:&quot;Calibri CE&quot;, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Panele podłogowe
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">7,32
            </td>
        </tr>
        <tr style="text-align:left;height:20px;">
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">1/2
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:none;border-left:none;padding:0 4.7px;">Pokój 1
            </td>
            <td style="text-align:center;font-family:&quot;Calibri CE&quot;, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Panele podłogowe
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">20,44
            </td>
        </tr>
        <tr style="text-align:left;height:20px;">
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">1/3
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:1px solid;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Pokój 2
            </td>
            <td style="text-align:center;font-family:&quot;Calibri CE&quot;, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Panele podłogowe
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">19,14
            </td>
        </tr>
        <tr style="text-align:left;height:20px;">
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">1/4
            </td>
            <td style="text-align:center;font-family:&quot;Calibri CE&quot;, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Łazienka
            </td>
            <td style="text-align:center;font-family:&quot;Calibri CE&quot;, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Płytki
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">10,16
            </td>
        </tr>
        <tr style="text-align:left;height:20px;">
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">1/5
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Pokój 3
            </td>
            <td style="text-align:center;font-family:&quot;Calibri CE&quot;, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Panele podłogowe
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">18,03
            </td>
        </tr>
        <tr style="text-align:left;height:20px;">
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">1/6
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Garderoba
            </td>
            <td style="text-align:center;font-family:&quot;Calibri CE&quot;, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">Panele podłogowe
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">12,69
            </td>
        </tr>
        <tr style="text-align:left;height:20px;">
            <td colspan="3" style="text-align:right;font-family:Calibri, sans-serif;vertical-align:bottom;border-top:1px solid;border-right:1px solid #000000;border-bottom:1px solid;border-left:1px solid;padding:0 4.7px;">
                <strong>SUMA
                </strong>
            </td>
            <td style="text-align:center;font-family:Calibri, sans-serif;vertical-align:bottom;border-top:none;border-right:1px solid;border-bottom:1px solid;border-left:none;padding:0 4.7px;">
                <strong>87,78
                </strong>
            </td>
        </tr>
    </table>
 </div>